src/Microsoft.Health.Fhir.Core/Features/Operations/Reindex/ReindexJobTask.cs

```diff
@@ -62,6 +69,22 @@ public async Task ExecuteAsync(ReindexJobRecord reindexJobRecord, WeakETag weakE
                 if (_reindexJobRecord.QueryList?.Count == 0)
                 {
                     // Build query based on new search params
+                    // Find supported, but not yet searchable params
+                    var notYetIndexedParams = _supportedSearchParameterDefinitionManager.GetSupportedButNotSearchableParams();
```

Comment:
I guess this is also the same as `_supportedSearchParameterDefinitionManager.AllSearchParameters.Where(p => !p.IsSearchable)`. All search params returned from _supportedSearchParameterDefinitionManager should be supported

Reply:
It is the same.  Just hiding it behind an interface.  I suppose we could simply remove this method.